docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,12 +1,20 @@
 == Installing Jenkins plugins
-Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them in Ansible playbook or using the Jenkins UI.
-
 NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
 
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them as a task in Ansible playbook or using the Jenkins UI.
+
+
+=== Using Jenkins UI
+Go to `Manage Jenkins -> Manage Plugins`. If the plugin is not available in the
+`Available` tab then it   can be installed by uploading an `.hpi` file in
+`Advanced -> Upload Plugin`.
+
 === Using Ansible Playbook
-The `jenkins_plugins` parameter can be overridden to include new plugins. The
-structure of a Jenkins plugin is as follows.
+//https://github.com/aerogear/digger-installer/blob/master/configure-buildfarm/defaults/main.yml[Digger Installer Ansible playbook] add this when the url is known (after renaming repos)
```

Comment:
Lets point the user to where they can update `jenkins_plugins` value. 
It's in 'aerogear-digger-installer -> configure-buildfarm -> defaults -> main.yml`. We should specify that they use the syntax shown by the default ones provided

Reply:
resolved